docs/jwst/associations/level3_asn_technical.rst

```diff
@@ -118,28 +126,28 @@ members *required*
   This is a list of the exposures to be used by the stage 3 processing
   tasks. This keyword is explained in detail in the next section.
 
+.. _asn-level3-members:
+
 ``members`` Keyword
 ^^^^^^^^^^^^^^^^^^^
 
-``members`` is a list of objects, each consisting of the following
-keywords
+``members`` is a list of dictionaries, one for each member exposure in the
+association. Each member has the following keywords.
 
 expname *required*
   The exposure file name
 
 exptype *required*
   Type of information represented by the exposure. Possible values are
 
-  * ``science`` *required*
-
-    The primary science exposures. There is usually more than one
+  * ``science``: The primary science exposures. There is usually more than one
     since stage 3 calibration involves combining multiple science
     exposures. However, at least one exposure in an association needs
     to be ``science``.
-    
-  * ``psf`` *optional*
 
-    Exposures that should be considered PSF references for
+  * ``background``: Exposures used for background subtraction.
```

Comment:
At level 3 this would only be for master bkg subtraction subtraction, right?

Reply:
That is what I thought. However, I left the wording more generic to be future-proof. Would preference be for more specificity?

Reply:
No, leave it generic for now.